compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1192,25 +1192,35 @@ static LogicalResult setRootConfig(func::FuncOp entryPointFn,
 }
 
 static SmallVector<int64_t>
-getDefaultDistributionTileSizes(TilingInterface op) {
-  unsigned numLoops = op.getLoopIteratorTypes().size();
-  // Set all the distribution tile sizes to zero if thread distribution is
-  // disabled.
-  if (clDisableDistribution) {
-    return SmallVector<int64_t>(numLoops, 0);
-  }
```

Comment:
We need to keep this functionality as we are using this flag to run some benchmarks. Is this preserved somewhere else?

Reply:
We still keep the functionality because the innermost function call has it. This is one of major reasons that I wanna remove special logics from this files as much as possible! Thanks for pointing it out, and I verified that we still have the support!